### ISSUE TITLE
FE | Profile | Direct Deposit - FE End to End Integration Work

### ISSUE THREAD

**Issue by adamwhitlock1** [department-of-veterans-affairs/va.gov-team#54896]
## Issue Description
Frontend work may need to be done to ensure E2E functionality for direct deposit Lighthouse migration is working as expected

Use below referenced ticket to incorporate the error code work that tom completed.  
- #58520

## Front end tasks
- Fraud flag has appropriate messaging on FE
- Profile being blocked due to deceased/fiduciary/incompetent flag works
- Direct Deposit can be updated
- Phone/Work/Address required fields present appropriate error messaging or guarding via FE

## Acceptance Criteria
- [x] E2E integration with Lighthouse Direct Deposit migration is done in staging
- [x] Unit tests pass
- [x] E2E tests cover current code and regression
- [x] End-to-end tests show 0 violations.
- [x] The data returned matches API response (for given user or scenario)
- [x] All axe checks pass
- [x] All color contrast checks pass
- [x] All zoom testing passes
- [x] All keyboard checks pass 
- [x] All screenreader checks pass

**Comment by adamwhitlock1** [comment]
Hadn't really gotten to this one since we were working on all the error codes and also the personal info unplanned work that had come up that kinda took some of my time away. I will hop on this first thing next sprint

**Comment by mtcA6** [comment]
@adamwhitlock1 can you make sure we [update this part](https://docs.google.com/spreadsheets/d/12gvlkGnwt206BBYY89gDIYG1AtD6j9uCYMr5x99Nzns/edit#gid=0&range=B18:B20) of the testing spreadsheet?

**Comment by adamwhitlock1** [comment]
I have completed all e2e test additions, have merged code to main and checked in staging to verify that the lighthouse endpoint is working as expected at this point.

The spreadsheet with error case testing has also been updated with the various ways to locally mock some of the edge cases along with a link to the specific code that provides the mock response.

@mtcA6 I think we can close this out, unless you see anything else that I am missing on this?
